### ISSUE TITLE
 emqttd_http can not use emq_auth_http?

### ISSUE THREAD

**Issue by CMTEST080** [emqtt/emqttd#739]
I tried to use emqttd_http and emq_auth_http. 
emq_auth_http got a status code 200 from an auth server. but emqttd_http replied a 401 error.

```
22:50:51.507 [error] HTTP Auth failure: username=pi4, reason={function_clause,[{emq_auth_http_cli,feedvar,[[{"clientid","%c"},{"username","%u"},{"password","%P"}],{mqtt_client,undefined,undefined,<<"pi4">>,undefined,undefined,undefined,0,undefined,undefined,undefined}],[{file,"src/emq_auth_http_cli.erl"},{line,46}]},{emq_auth_http,check,3,[{file,"src/emq_auth_http.erl"},{line,40}]},{emqttd_access_control,auth,3,[{file,"src/emqttd_access_control.erl"},{line,64}]},{emqttd_http,authorized,1,[{file,"src/emqttd_http.erl"},{line,136}]},{emqttd_http,handle_request,3,[{file,"src/emqttd_http.erl"},{line,47}]},{mochiweb_http,headers,5,[{file,"src/mochiweb_http.erl"},{line,75}]}]}
```

**Comment by turtleDeng** [comment]
hi
  Thank you, this is indeed a bug, we later modify the bug.

**Comment by CMTEST080** [comment]
Thanks!! I will try again after fixed that.

**Comment by CMTEST080** [comment]
Thanks! 
This issue was solved when I tried v2.0-rc.3.
